src/index.ts

```diff
@@ -1,3 +1,5 @@
 export { ias } from './third-party-tags/ias';
 export { permutive } from './third-party-tags/permutive';
 export { fbPixel } from './third-party-tags/facebook-pixel';
+export { twitter } from './third-party-tags/twitter-uwt';
+export { lotame } from './third-party-tags/lotame';
```

Comment:
This is definitely a `feat`!

Reply:
though `fix` is ok for 2 reasons
- feat was the previous PR here: https://github.com/guardian/commercial-core/pull/41 and this is a fix for that
- had to conclude to 1 prefix and fix sounded Ok for the 2 things and didn't want to raise 2 separate PRs. at the end the new release is made